pwiz_tools/Skyline/Model/Results/SpectrumFilterPair.cs

```diff
@@ -626,6 +632,11 @@ public override int GetHashCode()
             }
         }
 
+        public override string ToString() // For debug convenience
+        {
+            return $@"mz={TargetMz} w={FilterWidth} id={FilterId} heo={HighEnergyIonMobilityValueOffset}";

```

Comment:
I wonder whether we should only include "heo" if it's not null.

Reply:
Doesn't seem worth the bother for debug code